h2o-core/src/main/java/water/FailedNodeWatchdogExtension.java

```diff
@@ -24,7 +26,15 @@ public void printHelp() {
                         "    -watchdog_client\n" +
                         "          Same as the client except the that cluster is stopped when this client \n" +
                         "          disconnects from the rest of the cloud or the cloud is stopped when it doesn't \n" +
-                        "          hear heartbeat from the client for specified amount of time."
+                        "          hear heartbeat from the client for specified amount of time. \n" +
+                        "    -watchdog_client_connect_timeout\n" +
```

Comment:
I was thinking about this, i decided for new property for 2 reasons:
 - more fine-grained control
 - from my point of view, `watchdog_stop_without_client` should be higher than the other one since the clouding up can take some time ?